### PR TITLE
Update the operation guide for the end user authentication

### PR DESCRIPTION
**Pull request by lei-tang**
Update the operation guide for the end user authentication

### CODE REVIEW

File: content/help/ops/security/end-user-auth/index.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
 ---
-title: End-User Authentication
+title: End User Authentication
 description: What to do if end-user authentication doesn't work.
 weight: 80
 ---
 
-1. Check the policy, `principalBinding` should be set as `USE_ORIGIN`.
+With Istio, you can enable authenticating the end user. Currently, the end user credential supported by the Istio authentication policy is JWT.
+The following is a guide for troubleshooting the end user JWT authentication.
```
Comment (geeknoid) [inline content/help/ops/security/end-user-auth/index.md]: remove "the"

Comment (lei-tang) [reply content/help/ops/security/end-user-auth/index.md]: Done.

**Pull request merged**